TestingExamples/TestingExamples/RequestTests/NegativeTesting.cs

```diff
@@ -0,0 +1,35 @@
+﻿using NUnit.Framework;
+using RestSharp;
+using System.Net;
+using FluentAssertions;
+
+namespace ApiTestingExamples.RequestTests;
+
+public class NegativeTesting : BaseTest
+{
+    [Test]
+    [Property("Test", "TestNumber")]
+    public async Task NegativeTesting_NonExistentUser_ReturnsBadRequest()
+    {
+        await ApiRequestTest("testPath",
+            "nonExistentUser",
+            HttpStatusCode.BadRequest);
+    }
+
+    private async Task ApiRequestTest(string updatableInfo, string username, HttpStatusCode validationCode)
```

Comment:
Not every error response would necessarily have a useful error message attached, but it might be worth including as an optional param here. A "string errorMessage = null" would allow you to ignore it if/when you don't care, but also easily validate expected messages when relevant.

Reply:
added in a new test for validating error message and added it as a null'd variable by default so it's optional